pkg/logging/config.go

```diff
@@ -286,8 +285,8 @@ func load(config *Config) error {
 	viper.SetConfigName("logging")
 
 	// Set the path to look for the configurations file
-	viper.AddConfigPath("./" + configDir + "/config")
```

Comment:
I think it would be simpler to have changed line 12 to load `configdir` from an environment variable. If not present it can be `.onos` by default, so as not to break existing functionality.
Reading of this env var could be done in the `init()` function

Reply:
What existing behaviour does it break? I guess using a env-var is up to personal style so don't mind that part.

I didn't mention it earlier but the init function is (as far as I can see) a bit of an issue as well. As it contains code in its path that can panic. It is to my knowledge not possible to handle this panic.

Reply:
Fair point on the panic inside init(). 
As far as existing functionality is concerned - I mean existing uses expect a default value `.onos` if not ENV VAR is present